docs/definitions/class.md

```diff
@@ -0,0 +1,19 @@
+# Class
+
+In ML, a **class** (or category) is a possible value a discrete **output**
+variable can take. It is therefore used in the context of classification (or
+categorization) problems.
+
+Classes allow to differentiate (or to classify)
+the data – or a subset of the data – into a **finite** set of options.
+The different possible options an output variable can take are often
+referred to as classes.
+
+**Note**
+Classes and labels, although sometimes used indistinctively, do not share the
+exact same definition. Be sure to check the
+[`ml:label` definition](/definitions/label). A class is the possible value a
+`ml:label` may take, but a `ml:label` is not necessarily a class.
```

Comment:
Maybe adding a small example here could make this distinction clearer? E.g. (if I get it correct) 'colour' is a label and 'green', 'red', etc are classes?